app/src/renderer/components/common/TmModalNodeHalted.vue

```diff
@@ -0,0 +1,105 @@
+<template lang="pug">
+.ni-modal-error__wrapper
```

Comment:
are we using `ni-modal` still?

Reply:
https://github.com/cosmos/voyager/pull/967